google/cloud/security/inventory/pipeline_builder.py

```diff
@@ -90,10 +90,15 @@ def _get_api(self, api_name):
                 raise api_errors.ApiInitializationError(e)
 
             api_version = self.api_map.get(api_name).get('version')
-            if api_version is None:
-                api = api_class()
-            else:
-                api = api_class(version=api_version)
+            try:
```

Comment:
Sorry to do this, since you're touching this file can you resolve the pylint doc messages above?